### PR TITLE
feat: add PIWIK tracking code and define goal on start now button

### PR DESCRIPTION
**Pull request by geek-soft**
### JIRA link

https://companieshouse.atlassian.net/browse/LFA-1523

### Change description

Adds PIWIK tracking code to application.

In addition to default tracking it also tracks "Start now" button clicks as tracking goal.

### Work checklist

- [ ] Tests added where applicable
- [x] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -15,3 +15,5 @@ KAFKA_BROKER_ADDR=localhost:9092
 SC_TEAM_EMAIL='appeals.ch.fake+SC@gmail.com'
 NI_TEAM_EMAIL='appeals.ch.fake+NI@gmail.com'
 DEFAULT_TEAM_EMAIL='appeals.ch.fake+DEFAULT@gmail.com'
+
+CDN_HOST=chs-dev.internal:8080
```
Comment (n-r-b) [inline .env.local]: do we need to add this here if it's only used when the PIWIK_URL and PIWIK_SITE_ID are set?

File: src/utils/ConfigLoader.ts [hunk 2]
```diff
@@ -51,6 +52,15 @@ export const getExpressAppConfig = (directory: string) => (app: express.Applicat
                 visuallyHiddenText: accessibleName
             }
         }
+    };
+
+    const url = getEnv('PIWIK_URL');
+    const site = getEnv('PIWIK_SITE_ID');
```
Comment (n-r-b) [inline src/utils/ConfigLoader.ts]: is the PIWIK_SITE_ID the same for all apps?

File: src/views/landing.njk [hunk 3]
```diff
@@ -31,7 +31,8 @@
             text: 'Start now',
             isStartButton: true,
             attributes: {
-              id: 'start-now'
+              id: 'start-now',
```
Comment (n-r-b) [inline src/views/landing.njk]: The acceptance criteria mentions adding this to all continue like buttons, do we need to do this?

Comment (geek-soft) [reply src/views/landing.njk]: AC is wrong in that regard. It looks CH only sets goal on "start now" button

File: src/utils/ConfigLoader.ts (hunk 2 above)
Comment (geek-soft) [reply src/utils/ConfigLoader.ts]: Nope, every env has different site ID. For example we use 26 in Rebel and 10 in Staging.

File: .env.local (hunk 1 above)
Comment (geek-soft) [reply .env.local]: Good point, not strictly necessary but I know we will have CDN in use very soon.

File: src/utils/ConfigLoader.ts (hunk 2 above)
Comment (n-r-b) [reply src/utils/ConfigLoader.ts]: ok, i've already seen this has been configured in the global env properties 👍

**Pull request merged**